### PR TITLE
gifski: Fix checkver

### PR DESCRIPTION
**Pull request by h404bi**
From: https://github.com/ScoopInstaller/Main/issues/150

### CODE REVIEW

File: bucket/gifski.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "extract_dir": "win",
     "bin": "gifski.exe",
     "checkver": {
```
Comment (Ash258) [inline bucket/gifski.json]:
```suggestion
    "checkver": "For Windows.*?gifski-([\\d.]+)\\.zip",
```

Comment (h404bi) [reply bucket/gifski.json]: Are you sure? I can't see any tweaks after running `bin\formatjson.ps1`.

Comment (Ash258) [reply bucket/gifski.json]: https://github.com/lukesampson/scoop/blob/3e55a70971c5ff0d035daa54ca5dfab95dfaaa1d/bin/checkver.ps1#L109-L112

**Comment by rasa** [comment]
I ran `checkver -f` on this locally, and it updated fine.

**Pull request merged**